File: PracticaExamen/ObjetosTransferencia/TOProducto.vb
Public Class TOProducto
    Property Codigo As String
    Property Descripcion As String
    Property Precio As Integer

    Sub New()

    End Sub

    Sub New(cod As String, desc As String, pre As Integer)
        Me.Codigo = cod
        Me.Descripcion = desc
        Me.Precio = pre
    End Sub
End Class

File: Practicav2/ObjetosTransferencia/TOProducto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ObjetosTransferencia
{
    public class TOProducto
    {
        public String Codigo { get; set; }
        public String Descripcion { get; set; }
        public int Precio { get; set; }

        public TOProducto() { }

        public TOProducto(String cod, String des, int precio)
        {
            this.Codigo = cod;
            this.Descripcion = des;
            this.Precio = precio;
        }
    }
}
